integrations/static_analyzers/requirements.txt

```diff
@@ -1,2 +1,4 @@
 Flask-Shell2HTTP~=1.8.0
-gunicorn~=20.1.0
\ No newline at end of file
+gunicorn~=20.1.0
+flare-capa==2.0.0
```

Comment:
can you use ~= so we can get patches automatically?

Reply:
plus, can you check the status of the current available versions? I guess this is old

Reply:
Hey @mlodic . I've made the requested change.

Also, I looked into the packages. stringsifter is the same version but yes Capa is in v3.
I've looked into the tool. It seems that Capa v2 isnt working well with the latest rule set.
But just changing the dependency here wouldnt work because installing capa via pip doesnt setup the default library identification [signatures](https://github.com/mandiant/capa/tree/master/sigs)

I see there's already an issue for this, #708 . I can pick that up, I have a fair idea on what to do.

Reply:
sure, it would be great! you can pick up that issue! Meanwhile I accept this PR so we can close the first issue